src/functions/manager.c

```diff
@@ -20,19 +20,28 @@
 
 */
 
-/** \file */
+/** \file
+ * Singleton class.
```

Comment:
C doesn't have the concept of classes, so this is unclear. Is this meant to be a single-instance struct?